.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.3/apache-maven-3.6.3-bin.zip
```

Comment:
Could probably use a newer version like 3.8.1
https://github.com/dockstore/dockstore/blob/develop/.mvn/wrapper/maven-wrapper.properties

Is 3.6.3 on purpose?

Reply:
yes, i originally tried to use the newest version.  There were some differences in the generated pom.xml.  This version is the version from the image.

Reply:
Hmmm, please create a ticket for this as well and assign to me. 
I'm using Maven 3.8.1 locally and don't see any differences in the generated pom (Java 11.0.11)